### DISCUSSION TITLE
php7.2 环境 thinkphp5.1无法使用PHPExcel，同套代码在宝塔是没问题的

### DISCUSSION THREAD

**Discussion by tangmingze** [1Panel-dev/1Panel#5825] [General]
![image](https://github.com/user-attachments/assets/2a983c0f-dbb7-418b-bc9a-257c9b317daa)
你说不能用吧，其实又不是插件报错。就是最后的$PHPWriter->save("php://output");不行。
有遇到同样问题的兄弟么！？

**Comment by wan92hen** [comment]
检查下是不是开了网站防篡改呢。

**Reply by tangmingze** [reply]
已经解决了，是Excel2007需要模块php-zip的支持，php7.2扩展加zip后重建就解决问题了!
